src/nwbinspector/_formatting.py

```diff
@@ -218,30 +219,37 @@ def format_messages(self) -> list[str]:
 
 def format_messages(
     messages: list[InspectorMessage],
-    levels: list[str] = None,
+    levels: Optional[list[str]] = None,
     reverse: Optional[list[bool]] = None,
     detailed: bool = False,
 ) -> list[str]:
     """Print InspectorMessages in order specified by the organization structure."""
-    if levels is None:
-        levels = ["file_path", "importance"]
+    levels = levels or ["file_path", "importance"]
+
     message_formatter = MessageFormatter(messages=messages, levels=levels, reverse=reverse, detailed=detailed)
     formatted_messages = message_formatter.format_messages()
+
     return formatted_messages
 
 
-def print_to_console(formatted_messages: list[str]):
+def print_to_console(formatted_messages: list[str]) -> None:
     """Print report file contents to console."""
     sys.stdout.write(os.linesep * 2)
     for line in formatted_messages:
         sys.stdout.write(line + "\n")
 
+    return None
```

Comment:
You may notice this around the place now

I'm not super crazy about it, but it is more explicit (and Python says explicit is better than implicit...)

Could theoretically suppress all return checks, but it did catch some pretty useful ones elsewhere